### DISCUSSION TITLE
794

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#968] [Comments]
# 794

최근에 배기홍 대표님의 아래 글을 보았다.THE STARTUP BIBLE - 개발자도 회사의 조직원이다배기홍 대표님의 글은 항상 생각할거리를 던져주셔서 재밌게 보는데, 이번 글은 요즘의 불황시기에 겹쳐서 많이 언급되는 주제라 한번 더 생각하게 되었다.글은 개발자가 아닌 VC 분들이나 대표님 혹은 조직의 리더라면 충분히 이야기할 수 있는 주제인 것 같다.핵심으로 생각되는 문장은 다음과 같다.어떻게 생각해 보면 너무나 당연한데,우리 주변, 또는 우리 회사의 많은 개발자들이 그냥 본인들이 풀고 싶은 문제를 풀고, 본인들이 개발하고 싶은 기능을 개발하고, 기술적으로 어려운 문제를 코드로 미화하고 싶어 한다.단순히 엔지니어링 측면에서 보면 이게 대단하게 보일 수도 있지만,이런 결과물들이 회사의 경영진들이 설정한 ..

https://jojoldu.tistory.com/794

**Comment by OsoriAndOmori** [comment]
ㅎㅎ성숙한척 하고 싶은 중간연차입니다..
저연차부터 매번 기술부채 해결, 고도화를 하자고 상급자들에게 응애거렸던거 같은데.. 스스로를 돌아보게 되네요..

**Comment by jws1837** [comment]
진양철 회장님도 말했죠. "그게 돈이 됩니까?"  돈이 되는 일을 해서 돈을 받는 개발자가 비즈니스에 필요한 프로 개발자 같습니다.

**Comment by el-yun** [comment]
결국 공감대가 아닐까 싶습니다. 경영진의 생각, 제품을 만드는 실무진의 생각이 맞춰져야 모두가 공감하는 제품이 나오는게 아닐까요
